src/Materials/multiMaterial.ts

```diff
@@ -188,18 +188,20 @@ export class MultiMaterial extends Material {
 
         serializationObject.name = this.name;
         serializationObject.id = this.id;
+        serializationObject.uniqueId = this.uniqueId;

```

Comment:
Is there a way to avoid doing this for every material class?

Reply:
what we could do is do it at Material level (with id, name and tags) and simply call it from each child. That will be painful only once ;)

Reply:
Done!